src/theory/quantifiers/sygus/sygus_unif_rl.h

```diff
@@ -382,6 +387,8 @@ class SygusUnifRl : public SygusUnif
      * enumerated condiotion values
      */
     PointSeparator d_pt_sep;
+    /** random number generator */
+    std::default_random_engine d_rng;
```

Comment:
Remove this.

Reply:
See my comment above, this should indeed be removed since CVC4's RNG should be used.